.github/workflows/pipeline.yml

```
@@ -0,0 +1,68 @@
+name: CI-Checks
```

```suggestion
name: Linting and Tests
```

Just to be a bit more descriptive. I'm *pretty* these can have spaces in them.